### DISCUSSION TITLE
Should I expect any issues if I share an elasticsearch server between multiple instances?

### DISCUSSION THREAD

**Discussion by lullis** [mastodon/mastodon#24948] [Q&A]
Say that I have a few small instances and I don't want to set up one elasticserver for each. Could I share the same ES cluster among them?

**Comment by stevenjlm** [comment]
It won't cause problems if configured correctly, see [this ](https://docs.joinmastodon.org/admin/optional/elasticsearch/#config)section in the docs. Specifically,

> If you have multiple Mastodon servers on the same machine, and you are planning to use the same Elasticsearch installation for all of them, make sure that all of them have unique REDIS_NAMESPACE in their configurations, to differentiate the indices. If you need to override the prefix of the Elasticsearch indices, you can set ES_PREFIX directly.
